### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by Huluti**
- Lighter
- More efficient

### CODE REVIEW

File: src/data/js/jquery.min.js [hunk 1]
```diff
@@ -1 +1 @@
-jquery-1.9.1.min.js
\ No newline at end of file
+jquery-3.1.1.min.js
```
Comment (jonathonf) [inline src/data/js/jquery.min.js]: What is the reason for this change?

File: src/manjaro-welcome [hunk 2]
```diff
@@ -37,35 +37,27 @@ from gi.repository import WebKit, Gtk
...
 
 
 class WelcomeConfig(object):
     def __init__(self):
         # store our base architecture
-        if os.uname()[4] == 'x86_64':
-            self._arch = '64-bit'
-        else:
-            self._arch = '32-bit'
+        self._arch = '64-bit' if os.uname()[4] else '32-bit'
```
Comment (jonathonf) [inline src/manjaro-welcome]: Does this follow the convention for the rest of the code?

File: src/manjaro-welcome [hunk 3]
```diff
@@ -37,35 +37,27 @@ from gi.repository import WebKit, Gtk
...
+                    var, arg = line.rstrip().split('=')
+                    if var.startswith('DISTRIB_'):
+                        var = var[8:]
                     if arg.startswith('"') and arg.endswith('"'):
                         arg = arg[1:-1]
                     if arg:  # Ignore empty arguments
                         distinfo[var] = arg
-        except IOError as msg:
-            print(msg)
+    except (IOError, OSError) as msg:
```
Comment (jonathonf) [inline src/manjaro-welcome]: You're raising an exception instead of explicitly checking for the error condition. This is the wrong way round.

File: src/data/js/jquery.min.js (hunk 1 above)
Comment (Huluti) [inline src/data/js/jquery.min.js]: Because, it's the last version that is lighter, more efficient and works well.
No visible changes. Just under-the-hood improvements.

File: src/manjaro-welcome (hunk 3 above)
Comment (Huluti) [inline src/manjaro-welcome]: It's a very pythonic way to do that. 

> Raising exceptions is considered to be an acceptable, and Pythonic, approach for flow control in your program. Consider handling missing files with IOErrors, rather than if statements. In this situation, an IOError exception will be raised if the file exists but the user does not have read permissions.
http://www.pfinn.net/python-check-if-file-exists.html

File: src/manjaro-welcome (hunk 2 above)
Comment (Huluti) [inline src/manjaro-welcome]: Others parts of the code use the ternary operator.

**Comment by Huluti** [comment]
Not useful anymore.
